src/pages/learn/dm/Markdown_Tutorial/markdown.mdx

```diff
@@ -0,0 +1,257 @@
+---
+layout: ../../../../layouts/docs.astro
+---
+
+
+# __Markdown Tutorial Guide.__ 
+[@veritanuda](https://keybase.io/veritanuda)
+
+This guide will specifically explore the features of Markdown formatting throughout PlanarAlly. It is used in both [Notes](../../../../docs/game/notes) fields as well as the  [Chat](../../../../docs/game/chat) window. 
+
+Information in this guide was heavily sourced from the [Basic Syntax Guide](https://markdownguide.offshoot.io/basic-syntax/)
+
```

Comment:
Also something I would mention is that notes currently allow custom HTML, whereas for chat this is explicitly not allowed.

I plan to move this to by default not allow this in notes as well, but give users some option to enable it for either a specific user or campaign or something, as there are inherent security risks to allowing HTML and I want to be transparent with how the current setup works.

Reply:
Hmmm So are the features for notes and chat to align? Because I was using chats to hand out links to PDFs and things. And in Notes I was resolving images to my own Nextcloud instance.

If we are going to limit functionality it should be consistent either way